pkg/store/tree_test.go

```diff
@@ -20,14 +20,15 @@ import (
 	"testing"
 )
 
-const testJSON1 = `{"cont1a":{"cont2a":{"leaf2a":12,"leaf2b":1.14159,"leaf2c":"myvalue1a2c","leaf2d":1.14159,"leaf2e":12,"leaf2f":"YXMgYnl0ZSBhcnJheQ==","leaf2g":true},"leaf1a":"myvalue1a1a","list2a":[{"name":"First","rx-power":6,"tx-power":5},{"name":"Second","rx-power":8,"tx-power":7}]},"leafAtTopLevel":"WXY-1234"}`
+const testJSON1 = `{"cont1a":{"cont2a":{"leaf2a":12,"leaf2b":1.14159,"leaf2c":"myvalue1a2c","leaf2d":1.14159,"leaf2e":12,"leaf2f":"YXMgYnl0ZSBhcnJheQ==","leaf2g":true},"leaf1a":"myvalue1a1a","list2a":[{"name":"First","tx-power":5},{"name":"First","rx-power":6},{"name":"Second","tx-power":7},{"name":"Second","rx-power":8}],"list3a":[{"firstName":"First","secondName":"Second","tx-power":9}]},"leafAtTopLevel":"WXY-1234"}`
 
 const (
 	Test1Cont1ALeaf1a   = "/cont1a/leaf1a"
 	Test1Cont1AList2a1t = "/cont1a/list2a[name=First]/tx-power"
 	Test1Cont1AList2a1r = "/cont1a/list2a[name=First]/rx-power"
 	Test1Cont1AList2a2t = "/cont1a/list2a[name=Second]/tx-power"
 	Test1Cont1AList2a2r = "/cont1a/list2a[name=Second]/rx-power"
+	Test1Cont1AList3a1t = "/cont1a/list3a[firstName=First][secondName=Second]/tx-power"
```

Comment:
I would have expected the format should be like `/cont1a/list3a[firstName=First,secondName=Second]/tx-power`

Did you see the double bracketed format coming from somewhere else?

Reply:
Yes, I did, though I'm not sure I have the link handy. Let me try to dig it up.

Reply:
OK - then that's good enough for me - thanks